src/main/java/de/tum/in/www1/artemis/web/rest/ResultResource.java

```diff
@@ -164,16 +164,16 @@ public ResponseEntity<List<Feedback>> getResultDetails(@PathVariable Long partic
      */
     @GetMapping("participations/{participationId}/results/logs-available")
     @EnforceAtLeastTutor
-    public ResponseEntity<Map<Long, Boolean>> getLogsAvailabilityForResultsOfParticipation(@PathVariable long participationId) {
+    public ResponseEntity<Map<Long, String>> getLogsAvailabilityForResultsOfParticipation(@PathVariable long participationId) {
```

Comment:
maybe rename this so we know that we don't fetch a boolean now. Maybe getBuildJobIdsForResultsOfParticipation?

rename the endpoint maybe to `participations/{participationId}/results/build-job-ids`